classes/Mail.php

```diff
@@ -212,29 +210,19 @@ public static function Send($id_lang, $template, $subject, $template_vars, $to,
                     Tools::dieOrLog(Tools::displayError('Error: invalid SMTP server or SMTP port'), $die);
                     return false;
                 }
-                $connection = new Swift_Connection_SMTP(
-                    $configuration['PS_MAIL_SERVER'],
-                    $configuration['PS_MAIL_SMTP_PORT'],
-                    $configuration['PS_MAIL_SMTP_ENCRYPTION'] == 'ssl' ? Swift_Connection_SMTP::ENC_SSL : (($configuration['PS_MAIL_SMTP_ENCRYPTION'] == 'tls' ? Swift_Connection_SMTP::ENC_TLS : Swift_Connection_SMTP::ENC_OFF))
-                );
-                $connection->setTimeout(4);
-                if (!$connection) {
-                    return false;
-                }
-                if (!empty($configuration['PS_MAIL_USER'])) {
-                    $connection->setUsername($configuration['PS_MAIL_USER']);
-                }
-                if (!empty($configuration['PS_MAIL_PASSWD'])) {
-                    $connection->setPassword($configuration['PS_MAIL_PASSWD']);
-                }
+
+                $connection = Swift_SmtpTransport::newInstance($configuration['PS_MAIL_SERVER'], $configuration['PS_MAIL_SMTP_PORT'], $configuration['PS_MAIL_SMTP_ENCRYPTION'])
+                    ->setUsername($configuration['PS_MAIL_USER'])
+                    ->setPassword($configuration['PS_MAIL_PASSWD']);
+
             } else {
-                $connection = new Swift_Connection_NativeMail();
+                $connection = Swift_SmtpTransport::newInstance('localhost', 25);
```

Comment:
This code is reached when SMTP relay has been disabled. 
I think that we should be using the native `sendmail` binary here instead of the local SMTP server.
Let's see what the alternative is for swiftmailer `5.4.1`.

Btw, this is just my interpretation of the code. Don't hesitate to correct me if I'm wrong. :wink: